### ISSUE TITLE
Deprecated Google Sheets API Integration

### ISSUE THREAD

**Issue by squidgy617** [seiyria/deck.zone#87]
It appears that the application is calling out to a deprecated version of the Google Sheets API, resulting in an endless load time when trying to render cards from a Google Sheet.

**Comment by seiyria** [comment]
This is correct. If you have a PR that can fix it I'd love to have it but I don't have time to investigate myself. Google made this project infeasible to maintain long term.

**Comment by squidgy617** [comment]
I understand. I'm looking at it but not sure I'll get far with it. In any case, even without the Google Sheets integration this is still a very useful tool.

**Comment by fabform** [comment]
Handy to [find google sheet id](https://fabform.io/a/how-to-find-google-sheet-id)
